.evergreen/debian_package_build.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+#
+# Test mongocxx's Debian packaging scripts.
+#
+# Supported/used environment variables:
+#   IS_PATCH    If "true", this is an Evergreen patch build.
+
+set -o xtrace
+set -o errexit
+
+on_exit () {
+  if [ -e ./unstable-chroot/debootstrap/debootstrap.log ]; then
+    echo "Dumping debootstrap.log"
+    cat ./unstable-chroot/debootstrap/debootstrap.log
+  fi
+}
+trap on_exit EXIT
+
+if [ "${IS_PATCH}" = "true" ]; then
+  git diff HEAD -- . ':!debian' > ../upstream.patch
+  git diff HEAD -- debian > ../debian.patch
+  git clean -fdx
+  git reset --hard HEAD
+  if [ -s ../upstream.patch ]; then
+    [ -d debian/patches ] || mkdir debian/patches
+    mv ../upstream.patch debian/patches/
+    echo upstream.patch >> debian/patches/series
+    git add debian/patches/*
+    git commit -m 'Evergreen patch build - upstream changes'
+    git log -n1 -p
+  fi
+  if [ -s ../debian.patch ]; then
+    git apply --index ../debian.patch
+    git commit -m 'Evergreen patch build - Debian packaging changes'
+    git log -n1 -p
+  fi
+fi
+
+cd ..
+
+git clone https://salsa.debian.org/installer-team/debootstrap.git debootstrap.git
+export DEBOOTSTRAP_DIR=`pwd`/debootstrap.git
+sudo -E ./debootstrap.git/debootstrap unstable ./unstable-chroot/ http://cdn-aws.deb.debian.org/debian
+cp -a mongo-cxx-driver ./unstable-chroot/tmp/
+sudo chroot ./unstable-chroot /bin/bash -c "(set -o xtrace && \
+  apt-get install -y build-essential git-buildpackage fakeroot debhelper cmake curl ca-certificates libboost-dev libsasl2-dev libicu-dev doxygen && \
```

Comment:
To check, libicu and libsasl are libmongoc dependencies. But we need those because we are building the C driver from the debian package, right?

Reply:
We have libicu and libsasl listed as dependencies for building the C++ driver in the `debian/control` file.  I'd have to try the build after removing those to see if we really need them. In short, though, the `debian_package_build.sh` script needs to use the build dependencies specified in `debian/control`.